apps/arweave/src/ar.erl

```diff
@@ -498,6 +502,14 @@ parse_cli_args(["defragment_module", DefragModuleString | Rest], C) ->
 		io:format("~ndefragment_module value must be in the [number],[address] format.~n~n"),
 		erlang:halt()
 	end;
+parse_cli_args(["tls_cert_file", CertFilePath | Rest], C) ->
+    AbsCertFilePath = filename:absname(CertFilePath),
+    ar_util:assert_file_exists_and_readable(AbsCertFilePath),
```

Comment:
Rather than `throw` what about doing an `io:format` + `erlang:halt`? I think that's what we do elsewhere in the cli parsing (e.g. [here](https://github.com/ArweaveTeam/arweave/pull/505/files#diff-f148c1ce17ebe7e20d6eed136cc4f0c0e517b978a199351bb7e50aca826a8de4R420))